### PR TITLE
Add mw to review rotation and add some owner assignments

### PR DESCRIPTION
**Pull request by michaelwoerister**
I've also added a `debuginfo` group and fixed the ownership assignment for the `incremental` group. I hope I got the syntax right.

r? @davidtwco

### CODE REVIEW

File: triagebot.toml [hunk 1]
```diff
@@ -738,6 +739,9 @@ ast_lowering = [
     "@estebank",
     "@spastorino",
 ]
+debuginfo = [
+    "@michaelwoerister",
```
Comment (davidtwco) [inline triagebot.toml:743]:
```suggestion
    "@michaelwoerister",
    "@davidtwco"
```

Comment (davidtwco) [reply triagebot.toml:743]: Maybe @wesleywiser as well, having been active in wg-debugging?

**Review by davidtwco — APPROVED** [review]
r=me when ready

File: triagebot.toml (hunk 1 above)
Comment (michaelwoerister) [reply triagebot.toml:743]: Let's ask him 🙂 He certainly knows more about the MIR-side of debuginfo than I do (although I'd like to learn).

**Pull request merged**